modular_skyrat/modules/bsrpd/code/bsrpd.dm

```diff
@@ -44,8 +45,16 @@
 	else
 		. += "You cannot see the charge capacity."
 
+	. += span_notice("<b>Alt-Click</b> to toggle remote piping.")
+
+/obj/item/pipe_dispenser/bluespace/AltClick(mob/user)
+	. = ..()
+	toggle = !toggle
+	balloon_alert(user, "safety [toggle ? "off!" : "on"]")
```

Comment:
Although the gun safety metaphor basically makes sense, I'm not sure if people will intuitively understand what this actually means (even though the examine text explains it). Maybe "ranged safety"? I just don't want feedback to be confusing.